includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -335,8 +331,10 @@ public function get_upe_enabled_payment_method_ids() {
 	 * @return string[]
 	 */
 	public function get_upe_enabled_at_checkout_payment_method_ids( $order_id = null ) {
-		$available_method_ids = [];
-		foreach ( $this->get_upe_enabled_payment_method_ids() as $payment_method_id ) {
+		$capture                 = empty( $this->get_option( 'capture' ) ) || $this->get_option( 'capture' ) === 'yes';
```

Comment:
Updated the condition here because default is automatic (yes).

Reply:
Good catch!